src/pages/index.tsx

```diff
@@ -48,15 +49,20 @@ export default function HomePage({ schedule }) {
       return schedule;
     }
 
+    const fuse = new Fuse(schedule, {
+      keys: ['nama_lokasi_vaksinasi', 'wilayah', 'kecamatan', 'kelurahan'],
```

Comment:
@tibudiyanto  should we add weighted keys? `kecamatan` and `kelurahan` for example can be 
```suggestion
      keys: [
       {
         name: 'kecamatan',
         weight: 2,
       },
       {
         name: 'kelurahan',
         weight: 2,
       }
       // not sure if this works but default fuse default weight is 1 
      'nama_lokasi_vaksinasi',
      'wilayah'
      ],
```

Reply:
I think this is great